plugins/generator-1.20.4/neoforge-1.20.4/templates/elementinits/entities.java.ftl

```diff
@@ -68,6 +69,33 @@ public class ${JavaModName}Entities {
 		</#if>
 	</#list>
 
+	<#if entitiesWithInventory?size != 0>

```

Comment:
There is no need to rework this with attachments as this is an entity we control and thus can handle data in NBT logic (the methods you reworked but needen't to).

Block ME does not use attachments for this either, as there is no need to.

Reply:
Keep this capabilities?

Reply:
Of course. Just keep the storage the way it was implemented as there is no need to go overboard with attachments on this as there is a vanilla system for us to use

Reply:
:heavy_check_mark: Fixed